data/remnant/remnant ships.txt

```diff
@@ -1107,18 +1094,6 @@ ship "Penguin"
 		"scramble protection" 0.25
 		"ion resistance" 0.05
 		"scramble resistance" 0.05
-		"thrust" 4.2
-		"turn" 80.0
-		"reverse thrust" 4.2
-		"flare sprite" "effect/remnant flare/small"
-			"frame rate" 5
-		"flare sound" "plasma tiny"
-		"steering flare sprite" "effect/remnant flare/small"
-			"frame rate" 5
-		"steering flare sound" "plasma tiny"
-		"reverse flare sprite" "effect/remnant flare/small"
-			"frame rate" 5
-		"reverse flare sound" "plasma tiny"
```

Comment:
u didnt say smth bout a penguin and I dont think it should be nerfed, especially not in its one good perk of allowing good mobility